.github/workflows/release.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: release version
         id: theVersion
-        run: echo "::set-output name=theVersion::$(echo ${GITHUB_REF##*/})"
+        run: echo "theVersion=%(echo ${GITHUB_REF##*/})" >> $GITHUB_OUTPUT
```

Comment:
Why do you use `%(..)` for variable substitution ? I assume it thould be just

```
echo "theVersion=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
```

Reply:
Oops typo. Good catch. Will update tomorrow!